azure-pipelines.yml

```diff
@@ -27,6 +27,8 @@ variables:
     value: true
   - name: EnableReleaseOneLocBuild
     value: false
+  - name: GDN_ALLOW_ANALYZE_NO_TARGETS
+    value: true
```

Comment:
Where is this used?

Reply:
This was one of their recommendations earlier. I can remove it and see if it still works.

Reply:
Looking at the 1ES templates, this value of `GDN_ALLOW_ANALYZE_NO_TARGETS` is set to true already as an environment variable to the BinSkim task. I don't see anything that would read this as a variable, though. It is just hardcoded as true.